sample/src/main/res/layout/activity_main.xml

```diff
@@ -14,7 +14,8 @@
         app:layout_constraintBottom_toTopOf="@id/midGuideline"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toTopOf="parent"
+        android:contentDescription="@string/image_not_found" />
```

Comment:
This should be the contentDescription for this image. I know is just an sample app, but is better no information over wrong information

Reply:
I have removed that string